algorithms/benches/signature/schnorr_compressed.rs

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2019-2021 Aleo Systems Inc.
+// This file is part of the snarkVM library.
+
```

Comment:
We originally have a bench program for Schnorr. This file is the bench program for Schnorr compressed.